renderer/index.js

```diff
@@ -457,11 +445,11 @@ function onOpen (files) {
 
   // .torrent file = start downloading the torrent
   files.filter(isTorrent).forEach(function (torrentFile) {
-    addTorrent(torrentFile)
+    addTorrent(torrentFile.path)
```

Comment:
This will break on OS X where `onOpen` gets called from these event handlers in `main/index.js`: 

```js
app.on('open-file', onOpen)
app.on('open-url', onOpen)
```

`onOpen` also gets called with a string when launching a duplicate copy of the app.